integration/bun/index.ts

```diff
@@ -5,12 +5,19 @@ const replicate = new Replicate({
 });
 
 export default async function main() {
-  return await replicate.run(
-    "replicate/hello-world:5c7d5dc6dd8bf75c1acaa8565735e7986bc5b66206b55cca93cb72c9bf15ccaa",
-    {
-      input: {
-        text: "Brünnhilde Bun",
-      },
+  const model = "meta/llama-2-70b-chat";
```

Comment:
Using a real language model significantly increases the flakiness. We should switch to a test model, increase test timeout, and/or add retry logic.

Reply:
I think for the moment let's use `replicate/canary` as it's fast, output is known and consistent and exercises the right streaming APIs.

https://github.com/replicate/cog-examples/tree/main/canary

I think this week we could pair on standing up a dedicated test model that covers the as much of the prediction API as possible, including file uploads -> streaming response.